ci-operator/step-registry/hypershift/openstack/create/hostedcluster/hypershift-openstack-create-hostedcluster-commands.sh

```diff
@@ -30,14 +33,34 @@ COMMAND=(
   --openstack-external-network-id "${OPENSTACK_EXTERNAL_NETWORK_ID}"
   --openstack-node-flavor "${OPENSTACK_COMPUTE_FLAVOR}"
   --openstack-node-image-name "${RHCOS_IMAGE_NAME}"
+  --node-upgrade-type InPlace
   --base-domain "${HYPERSHIFT_BASE_DOMAIN}"
   --control-plane-availability-policy "${HYPERSHIFT_CP_AVAILABILITY_POLICY}"
   --infra-availability-policy "${HYPERSHIFT_INFRA_AVAILABILITY_POLICY}"
-  --pull-secret=/etc/ci-pull-credentials/.dockerconfigjson
+  --pull-secret=/tmp/global-pull-secret.json
   --release-image "${RELEASE_IMAGE}"
   --annotations=hypershift.openshift.io/skip-release-image-validation=true
 )
 
+if [ "${NFV_NODEPOOLS}" == "true" ]; then
+  if test -f "${SHARED_DIR}/OPENSTACK_DPDK_NETWORK_ID"; then
+    OPENSTACK_DPDK_NETWORK_ID="$(<"${SHARED_DIR}/OPENSTACK_DPDK_NETWORK_ID")"
+    COMMAND+=("--openstack-node-additional-port=network-id:$OPENSTACK_DPDK_NETWORK_ID,disable-port-security:true")
+  fi
+  if test -f "${SHARED_DIR}/OPENSTACK_SRIOV_NETWORK_ID"; then
+    OPENSTACK_SRIOV_NETWORK_ID="$(<"${SHARED_DIR}/OPENSTACK_SRIOV_NETWORK_ID")"
+    COMMAND+=("--openstack-node-additional-port=network-id:$OPENSTACK_SRIOV_NETWORK_ID,vnic-type:direct,disable-port-security:true")
+  fi
+  # Use private credentials to pull CNF images for SR-IOV network operator
+  # Credentials are in shiftstack vault: shiftstack-secrets/quay-openshift-credentials
+  QUAY_USERNAME=$(cat /var/run/quay-openshift-credentials/quay_username)
+  QUAY_PASSWORD=$(cat /var/run/quay-openshift-credentials/quay_password)
+  QUAY_AUTH=$(echo -n "${QUAY_USERNAME}:${QUAY_PASSWORD}" | base64 -w 0)
+  curl -s -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /tmp/jq && chmod +x /tmp/jq
```

Comment:
FYI this has moved to `https://github.com/jqlang/jq/releases` and the latest release in 1.7.1. We probably want to update to that rather than using a 6 year old release?